manifests/w/WorldTimeCli/WorldTimeCli/1.0.0.0/WorldTimeCli.WorldTimeCli.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: WorldTimeCli.WorldTimeCli
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: WorldTimeCli
+PackageName: WorldTimeCli
+License: MIT License
+Copyright: wtc

```

Comment:
This is not a well-formed copyright. Please update this to include the full copyright information, including the year of the copyright